### PR TITLE
PEP 557: Fix typo (expicitely -> explicitly)

### PR DESCRIPTION
**Pull request by chris-martin**

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately we couldn't find an account corresponding to your GitHub username on [bugs.python.org](https://bugs.python.org/) (b.p.o) to verify you have signed the CLA (this might be simply due to a missing "GitHub Name" entry in your b.p.o account settings). This is necessary for legal reasons before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for your contribution, we look forward to reviewing it!

**Comment by Rosuav** [comment]
Thanks for catching that! I'm going to merge this despite the bot's concerns, since it's a trivial change; but if you want to contribute further in the future, signing the CLA will make things smoother.

File: pep-0557.rst [hunk 1]
```diff
@@ -237,7 +237,7 @@ The parameters to ``dataclass`` are:
   carefully.
 
   If a class already has an explicitly defined ``__hash__`` the
-  behavior when adding ``__hash__`` is modified.  An expicitely
+  behavior when adding ``__hash__`` is modified.  An expicitly
```
Comment (Jamim) [inline pep-0557.rst]: `expicitly`? :man_facepalming:

**Comment by Jamim** [comment]
Hello @Rosuav,
You merged a PR that changes `expicitely` to `expicitly` :sweat_smile:
It must be `explicitly`, not `expicitly`.

**Comment by Rosuav** [comment]
LOL, I saw half the error and didn't see the other half. Ah well.

**Comment by chris-martin** [comment]
Oh hahah I'm sorry :laughing:

**Pull request merged**